takoyaki/src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.TalonFXInvertType;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+
+public class Intake extends SubsystemBase {
+  private WPI_TalonFX falcon = new WPI_TalonFX(Constants.kIntake.FALCON_MOTOR_ID);
+  private DoubleSolenoid solenoid;
+
+  public Intake() {
+    falcon.configFactoryDefault();
+
+    falcon.setInverted(TalonFXInvertType.Clockwise);
+    
+    solenoid = new DoubleSolenoid(PneumaticsModuleType.CTREPCM, Constants.kIntake.SOLENOID_FRONT, Constants.kIntake.SOLENOID_BACK);
+
+    solenoid.set(DoubleSolenoid.Value.kOff);
+    solenoid.set(DoubleSolenoid.Value.kReverse);
+  }
+
+  
+  public void startIntake() {
+    setMotorPower(Constants.kIntake.INTAKE_SPEED);
+  }
+
+  public void stopIntake() {
+    setMotorPower(0);
+  }
+
+  public void startReverse() {
+    setMotorPower(-Constants.kIntake.INTAKE_SPEED);
+  }
+
+  public void setMotorPower(double velocity) {
```

Comment:
if we do setMotorPower here, we should do setMotorPower for all the subsystems that do open loop stuff (hopper)